.github/workflows/ci.yml

```
@@ -1,44 +1,39 @@
 name: CI
-
 on:
   pull_request:
   push:
     branches:
       - master
-
 jobs:
   test:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
+    name: OTP ${{matrix.otp}} / Elixir ${{matrix.elixir}}
     env:
       MIX_ENV: test
     strategy:
       fail-fast: false
       matrix:
         include:
-          - pair:
-              elixir: 1.11.4
-              otp: 23.2
-            lint: lint
+        - elixir: '1.11.4'
+          otp: '23.3'
+        - elixir: '1.14.5'
+          otp: '24.3'
+        - elixir: '1.15.0-rc.2'
+          otp: '26.0.1'
```

@josevalim can this be 1.15 no?

```suggestion
        - elixir: '1.15.0'
          otp: '26.0.1'
```